go-controller/pkg/ovn/address_set/fake_address_set.go

```diff
@@ -248,6 +249,13 @@ func (f *FakeAddressSetFactory) EventuallyExpectAddressSetWithAddresses(dbIDsOrN
 	}).Should(gomega.Succeed())
 }
 
+func (f *FakeAddressSetFactory) EventuallyExpectAddressSetWithAddressesFailure(dbIDsOrNsName any, addresses []string) {
```

Comment:
should we maybe use ginkgo helper here ?

Reply:
wdym please ?
but imo if it is nit, i would prefer to keep it simple, unless there is something neat and not complex

Reply:
since you're using an expectation from within an helper, this will mess up the stack trace of the failure.

You either use `GinkgoHelper` *or* set an offset on the caller - https://pkg.go.dev/github.com/onsi/gomega#ConsistentlyWithOffset